src/components/LabelBasicInput/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Label from '../styled-components/Label';
+import BasicInput from '../styled-components/BasicInput';
+
+const LabelBasicInput = (props: any) => {
```

Comment:
```suggestion
const LabelBasicInput = (props: LabelBasicInputProps) => {
```
any 보다는 엄격한 type 선언은 어떨까요?

Reply:
이것역시 동준님과 협의하여 반영하도록 하겠습니다.